ssv/docs/README.md

```diff
@@ -0,0 +1,54 @@
+# Documentation
+
+
+<p align="center",float="left">
+<img src="./simple_class_diagram.drawio.png", width="70%" height="10%">
+</p>
+
+## Network
+
+`Network` is an interface that allows the broadcasting of a message to the P2P network through its `Broadcast` method.
+
+## BeaconNode
+
+`BeaconNode` is an interface that allows the node to fetch duty data (e.g. an Attestation vote and a block to be proposed) and to submit a signed data (e.g. a signed Attestation vote and a signed block).
+
+It also provides a method to get the blockchain domain (a sequence of bytes used in signatures that identifies the blockchain) and it holds a `BeaconNetwork` that provides information about the blockchain such as the current slot and fork version.
+
+## Runner
+
+`Runner` is an interface that provides methods to execute a duty following the SSV protocol. It has 6 implementations:
+- `CommitteeRunner`
+- `ProposerRunner`
+- `AggregatorRunner`
+- `SyncCommitteeContributionRunner`
+- `ValidatorRegistrationRunner`
+- `VoluntaryExitRunner`
+
+Each implementation of `Runner` performs its associated duty type, as suggested by their names. A notable difference is that a `CommitteeRunner` object will only execute one `CommitteeDuty` throughout its life. Other `CommitteeRunner` objects will be created to execute upcoming `CommitteeDuties`. For the other runners, they will execute all duties of their associated types during its life.
```

Comment:
Perhaps clarify why the Runners differ in this behavior of
- some execute all the duties
- others are just 1-time performers